packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Primitives/PropertyWireInformation.cs

```diff
@@ -29,6 +31,7 @@ internal PropertyWireInformation(InputModelProperty inputModelProperty)
             SerializationFormat = CodeModelPlugin.Instance.TypeFactory.GetSerializationFormat(inputModelProperty.Type);
             IsRequired = inputModelProperty.IsRequired;
             IsReadOnly = inputModelProperty.IsReadOnly;
+            IsDiscriminator = inputModelProperty.IsDiscriminator;
```

Comment:
I believe whether the property is a discriminator is also an important part of the wire info of the property.